logistics_project/apps/malawi/models.py

```diff
@@ -13,5 +14,67 @@ class Organization(models.Model):
     def __unicode__(self):
         return self.name
 
+
+class RefrigeratorMalfunction(models.Model):
+    REASON_NO_GAS = '1'
+    REASON_POWER_FAILURE = '2'
+    REASON_FRIDGE_BREAKDOWN = '3'
+
+    REASONS = (
+        REASON_NO_GAS,
+        REASON_POWER_FAILURE,
+        REASON_FRIDGE_BREAKDOWN,
+    )
+
+    # A reference to the facility with the malfunction
+    supply_point = models.ForeignKey('logistics.SupplyPoint', db_index=True, related_name='+')
+
+    # Timestamp when the facility user reported the malfunction
+    reported_on = models.DateTimeField(db_index=True)
+
+    # Takes the value of one of the REASON_* constants above
+    malfunction_reason = models.CharField(max_length=1)
```

Comment:
i've been avoiding using `choices` for the reasons outlined [here](https://github.com/dimagi/commcare-hq/pull/12286#discussion_r69177555)